package.json

```diff
@@ -5,12 +5,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "xo",
-    "pretest": "npm run lint",
-    "test": "mocha",
-    "precoverage": "rimraf .nyc_output coverage",
-    "coverage": "nyc npm run test",
-    "preci": "echo start CI ........",
-    "ci": "npm run coverage"
+    "test": "jest"
```

Comment:
May I ask why should we consider linting as part of the testing?

On CI this change will force us to remove the explicit linting step because it will run implicitly as part of the tests. If you ask me that's worse developer experience. The CI will be telling me the tests failed but that's not true!

Reply:
We invoke the linting command as a suffix for the test command with a quality assurance objective, especially during the development locally.

I thought we were still with TravisCI here, so with the Github Actions, you can skip this suggestion.